judge/views/problem.py

```diff
@@ -406,11 +409,10 @@ def get_context_data(self, **kwargs):
         context['search_query'] = self.search_query
         context['completed_problem_ids'] = self.get_completed_problems()
         context['attempted_problems'] = self.get_attempted_problems()
-
-        context.update(self.get_sort_paginate_context())
-        context.update(self.get_sort_context())
-        context['hot_problems'] = hot_problems(timedelta(days=1), settings.DMOJ_PROBLEM_HOT_PROBLEM_COUNT)
+        context['hot_problems'] = self.get_hot_problems()
         context['point_start'], context['point_end'], context['point_values'] = self.get_noui_slider_points()
+        context.update(self.get_sort_context())
+        context.update(self.get_sort_paginate_context())
```

Comment:
Just curious, why do you need to move this line to this?

Reply:
Just for aesthetics